mizar/common/kubernetes_util.py

```diff
@@ -31,7 +31,16 @@ def build_label_filter(label_dict):
         str_list.pop()
     return "".join(str_list)
 
+def list_namespaces_by_labels(label_dict):
+    label_filter = build_label_filter(label_dict)
+    v1 = client.CoreV1Api()
```

Comment:
Please use object as the param: 
https://github.com/CentaurusInfra/mizar/blob/dev-next/mizar/common/common.py#L90

Reply:
Fixed